code/src/UI/Controls/ImageEx.cs

```diff
@@ -50,7 +73,7 @@ private UIElement CreateFromBitmap()
             var image = new Image
             {
                 Source = bitmap,
-                Stretch = Stretch
+                Stretch = Stretch,
```

Comment:
Having commas on the end of the last line is recommended as it means that adding a line after it means you only have to add the new line, not also change the last old line too.